src/app/pages/FiatOnrampPage/index.tsx

```diff
@@ -14,27 +14,33 @@ import { networkActions } from '../../state/network'
 import { CheckBox } from 'grommet/es6/components/CheckBox'
 import { selectThirdPartyAcknowledged } from './slice/selectors'
 import { fiatOnrampActions } from './slice'
+import { useContext } from 'react'
+import { ResponsiveContext } from 'grommet/es6/contexts/ResponsiveContext'
+import { ShareRounded } from 'grommet-icons/es6/icons/ShareRounded'
+import { Paragraph } from 'grommet/es6/components/Paragraph'
 
-function HeaderLayout(props: { children?: React.ReactNode }) {
+function Layout(props: { children?: React.ReactNode }) {
   const { t } = useTranslation()
   return (
     <Box
       round="5px"
       border={{ color: 'background-front-border', size: '1px' }}
       background="background-front"
       pad="medium"
+      alignSelf="center"
+      width="651px" // Padding + Transak threshold for >mobile layout
```

Comment:
`max-width` maybe? Kind of looks weird to me, setting a width. Not sure even extension has that much width available.